### PR TITLE
added styling Button

### PR DESCRIPTION
**Pull request by sanjitsarkar**
added styling for text button and icon button

### CODE REVIEW

File: css/components/button/index.css [hunk 1]
```diff
@@ -0,0 +1,145 @@
+button, input[type=submit] {
+    cursor: pointer;
+    outline: none;
+    border: none;
+    padding: 0.5rem 0.6rem;
+    border-radius: 0.3rem;
+    background-color: #119195;
+    color: rgb(255, 255, 255);
```
Comment (rutu14) [inline css/components/button/index.css:8]: Assign variables to colors

File: css/util/color/index.css [hunk 2]
```diff
@@ -0,0 +1,41 @@
+
+.bg-primary {
+    background-color: var(--primary-color)  !important;
+}
```
Comment (rutu14) [inline css/util/color/index.css:4]: Avoid using important

**Review by rutu14 — APPROVED** [review]

File: css/components/button/index.css (hunk 1 above)
Comment (sanjitsarkar) [reply css/components/button/index.css:8]: okay

File: css/util/color/index.css (hunk 2 above)
Comment (sanjitsarkar) [reply css/util/color/index.css:4]: okay

**Review by humzahasan — APPROVED** [review]
Looks good to merge!

**Pull request merged**